### PR TITLE
Update affiliation for Elastic employees

### PR DESCRIPTION
**Pull request by AlexanderWert**

### CODE REVIEW

File: developers_affiliations1.txt [hunk 1]
```diff
@@ -2946,6 +2946,8 @@ AndrzejOlender: andrzej!olender.me
 	Yakudo from 2017-08-01 until 2021-01-01
 	Saint-Gobain Sekurit from 2021-01-01 until 2021-11-01
 	eSky from 2021-11-01
+andrzej-stencel: andrzej-stencel!users.noreply.github.com, andrzej.stencel!elastic.co
```
Comment (michalpristas) [inline developers_affiliations1.txt:2949]: cc @andrzej-stencel

File: developers_affiliations7.txt [hunk 2]
```diff
@@ -4949,9 +4949,10 @@ rogeralsing: rogeralsing!users.noreply.github.com
 	Akka.NET from 2016-05-01 until 2016-12-01
 	Betsson Group from 2016-12-01 until 2017-08-01
 	Bonliva from 2017-08-01
-rogercoll: rogercoll!protonmail.com, rogercoll!users.noreply.github.com
+rogercoll: rogercoll!protonmail.com, rogercoll!users.noreply.github.com, roger.coll!elastic.co
```
Comment (AlexanderWert) [inline developers_affiliations7.txt:4952]: cc @rogercoll

**Review by andrzej-stencel — APPROVED** [review]
Approved!

**Review by rogercoll — APPROVED** [review]
Thanks! LGTM

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**